automations/assign_reviewers_by_directory/assign-reviewers-by-directory.cm

```diff
@@ -0,0 +1,20 @@
+# -*- mode: yaml -*-
+
+manifest:
+  version: 1.0
+
+automations:
+  review_ui:
+    # Triggered for PRs that contain any changes to JavaScript files inside the `src/ui` directory.
+    if:
+      - {{ files | match(regex=r/src\/ui\/.*\.js$/) | some}}
+    # Add a specified user and team as reviewers.
+    # Customize the reviewers to match your organization
+    run:
+      - action: add-reviewers@v1
+        args:
+          reviewers: [my-teamate, my-organization/ui-team]
+    - action: add-comment@v1
+        args:
+          comment: |
+            The UI team has automatically been added for review because this PR contains changes to components inside `/src/ui`
```

Comment:
this should be indent >> (2 spaces to the right)

Reply:
Great catch, I thought I had updated this mistake in this branch, but I guess not.